### DISCUSSION TITLE
how to change media device target?

### DISCUSSION THREAD

**Discussion by Nemra1** [mynaparrot/plugNmeet-server#214] [General]
first at all thank you for develop this perfect tool its help a lot..
My question is how to change camera source or mice source daring stream because i cant find any guide about that..

**Comment by jibon57** [comment]
You'll get small arrow in camera/webcam icon. By click on that arrow you'll see all your media devices. You can change to any device.

**Reply by Nemra1** [reply]
> You'll get small arrow in camera/webcam icon. By click on that arrow you'll see all your media devices. You can change to any device.

> You'll get small arrow in camera/webcam icon. By click on that arrow you'll see all your media devices. You can change to any device.

i mean in pure JavaScript one to many broadcast I'm trying to develop broadcaster function and watcher function 
`var JoinBroadcaster = function (data) {
createRoom
}`
`var JoinViewer = function (e) {
join room
}`
